internal/controllers/clusterextension_controller_test.go

```diff
@@ -63,7 +64,7 @@ func TestClusterExtensionNonExistentPackage(t *testing.T) {
 	t.Log("By running reconcile")
 	res, err := reconciler.Reconcile(ctx, ctrl.Request{NamespacedName: extKey})
 	require.Equal(t, ctrl.Result{}, res)
-	require.EqualError(t, err, fmt.Sprintf("no package %q found", pkgName))
+	require.EqualError(t, err, fmt.Errorf("no package \"%v\" found", pkgName).Error())
```

Comment:
Nit: Why was `fmt.Sprintf()` insufficient here?

Reply:
reverted to sprintf everywhere now